src/main/target/system_stm32f7xx.c

```diff
@@ -338,7 +338,10 @@ void SystemInit(void)
     const uint32_t vtorOffset = (uint32_t) &isr_vector_table_base;
 #define VTOR_OFFSET_ALIGNMENT 0x200
 #define VTOR_OFFSET_MASK (VTOR_OFFSET_ALIGNMENT - 1)
-    STATIC_ASSERT((vtorOffset % VTOR_OFFSET_MASK) == 0, isr_vector_table_base_is_not_512_byte_aligned);
+    if (vtorOffset % VTOR_OFFSET_MASK != 0) {
+        // ISR vector table base is not 512 byte aligned
+        while (1);
+    }
```

Comment:
This is causing F7's to wedge (tested on MATEKF722). Seems like the test for alignment should be based on `VTOR_OFFSET_ALIGNMENT`.  I'm not sure why the test is against 1 less and using `#define VTOR_OFFSET_MASK (VTOR_OFFSET_ALIGNMENT - 1)`.  I tested with using just `VTOR_OFFSET_ALIGNMENT` and it worked normally.

Reply:
Oops. I meant to test that this was working, after discovering that the previous test with `STATIC_ASSERT` had just yielded a nop in either case.